src/index.html

```diff
@@ -20,8 +21,709 @@
+      </div>
+
+      <div class="card">
+        <a
+          href="#"
+          class="card__image"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </a>
+
+        <h2
+          class="card__title"
+          href="#"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="rating-stars">
+            <div class="stars stars--0 stars--hidden">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars stars--1 stars--hidden">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars stars--2 stars--hidden">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars stars--3 stars--hidden">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars stars--5 stars--hidden">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <span class="rating-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span class="price-text">Price:</span>
+          <span class="price-value">$2,199</span>
+        </div>
+
+        <button class="card__buy-button">buy</button>
+      </div>
+      <div class="card">
+        <a
+          href="#"
+          class="card__image"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </a>
+
+        <h2
+          class="card__title"
+          href="#"
```

Comment:
The `h2` element should not have an `href` attribute. Consider removing it as this attribute is not valid for `h2` elements.